dags/veda_data_pipeline/veda_transfer_pipeline.py

```diff
@@ -37,7 +38,7 @@
     "filename_regex": "<file_regex>",
     "target_bucket": "<target_bucket>",
     "collection": "<collection-id>",
-    "cogify": "true|false",
+    "cogify": Param(type="boolean"),
```

Comment:
Can we give cogify an explicit default value 
```python
"cogify": Param(default=False, type="boolean")
```
I know this is out of scope, but can we add an explicit type for dry_run as well?

Reply:
https://github.com/NASA-IMPACT/veda-data-airflow/pull/301/commits/cd62aa92a41e7b18baf5886aee2613d946c8c984